okhttp-clients/src/main/java/com/palantir/remoting3/okhttp/ForwardingCall.java

```diff
@@ -63,7 +63,17 @@ public boolean isCanceled() {
     }
 
     @Override
-    public ForwardingCall clone() {
-        return new ForwardingCall(delegate.clone());
+    public Call clone() {
```

Comment:
What's calling this?

Reply:
OkHttp defines the API:
```
  /**
   * Create a new, identical call to this one which can be enqueued or executed even if this call
   * has already been.
   */
  Call clone();
```

and we call it in AsyncQosIoExceptionHandler